### ISSUE TITLE
Length of a variant question: Homlen, brekapoints vs. Consensus len.

### ISSUE THREAD

**Issue by fritzsedlazeck** [dellytools/delly#144]
Hi,
I got confused about homlen vs. breakpoints vs. Consensus length. 
Someone asked me about this variant (below). Here the breakpoints  suggest a 172kp region. However, the consensus and HOMLEN suggest a 74bp. So how do I need to interpret that?
Is it a 172kbp deletion with a 74 bp sequence that is repeated almost 4000times?

Thanks
Fritz

10 17443837 DEL00048980 A . PASS CE=1.86599;CHR2=10;CIEND=-32,32;CIPOS=-32,32;CONSENSUS=TCCCTTCCTTACACCTTGTACAAAAATTAATTCAAGATGGATTAGAGACTTCAATGTTAGACCTAAAACCATAAAAAACCCTAGAAGAAAACCTAGGCAATACCATTCAGGACATAGGCATAAGCAAAGACCTCACGACTAAAACATAAAGCAATGGCAACAAAAGCCAAAATTGACAAATGGGATCTA;CT=3to5;END=17616973;HOMLEN=47;INSLEN=0;MAPQ=22;PE=8;PRECISE;SR=10;SRQ=1;SVMETHOD=EMBL.DELLYv0.7.7;SVTYPE=DEL GT:GL:GQ:FT:RCL:RC:RCR:CN:DR:DV:RR:RV 0/1:-31.3808,0,-42.2763:10000:PASS:28846:56789:29159:2:47:6:19:15

**Comment by tobiasrausch** [comment]
Hi Fritz,

The consensus sequence is the assembled alternative haplotype (supporting the SV). If you BLAT that sequence you sometimes get the same breakpoints as Delly but BLAT of course strongly favors single locus alignments even if the percent identity is <100%. If you split the above consensus in a prefix and suffix portion then BLAT returns the same matches as Delly at 100% identity:
Prefix: TCCCTTCCTTACACCTTGTACAAAAATTAATTCAAGATGGATTAGAGACTTCAATGTTAGACCTAAAACCATAAAAAACCCTAGAAGAAAACCTAGGCAATACCATTCAGGACATAGG
Suffix
ATAAGCAAAGACCTCACGACTAAAACATAAAGCAATGGCAACAAAAGCCAAAATTGACAAATGGGATCTA

In short, the SV call is indeed a deletion-type SV that spans 172kbp but as you can see from BLAT-ing the whole consensus sequence the percent identity shift is minor to full-length alignments at a single genomic locus, so clearly a borderline SV call at low mapping quality (INFO:MAPQ=22). HOMLEN also underlines that the breakpoints are repetitive. A breakpoint homology length of 47bp suggests a very high similarity between the sequence at the left and right breakpoint and thus, that's a difficult to call region for illumina data.

Best, Tobias
